addons/mail/static/src/core/common/composer.js

```diff
@@ -64,6 +74,10 @@ export class Composer extends Component {
     static template = "mail.Composer";
 
     setup() {
+        this.KEYBOARD = {
+            NONE: "None",
```

Comment:
I would keep it, because `KEYBOARD` contains all states in state machine with showing of keyboard. `undefined` or `null` doesn't define it, so this makes it harder to grep and see where it's used (and also not easy to spot at a glance)